e2e/jail/jail_test.go

```diff
@@ -146,10 +146,11 @@ func (s *JailTestSuite) TestEventSignal() {
 	opCompletedSuccessfully := make(chan struct{}, 1)
 
 	// replace transaction notification handler
+	defer signal.ResetDefaultNodeNotificationHandler()
 	signal.SetDefaultNodeNotificationHandler(func(jsonEvent string) {
 		var envelope signal.Envelope
-		err = json.Unmarshal([]byte(jsonEvent), &envelope)
-		s.NoError(err)
+		er := json.Unmarshal([]byte(jsonEvent), &envelope)
```

Comment:
where the use of `err` part of the panic cause?

Reply:
Yeah, why do we change that? It should be affecting anything and `er` looks like a typo.

Reply:
@influx6 There is the datarace at writing to err var). 
err was defined upper. Than signal is called from jail.

@adambabik I don't see any reasons to affecting anything. SetDefaultNodeNotificationHandler is used only in tests.

Reply:
I see. Ok, let's at least change it to something meaningful like `unmarshalErr` or something.

Reply:
@adambabik just renamed.